Source/WTF/wtf/text/ASCIILiteral.h

```diff
@@ -108,6 +120,8 @@ class ASCIILiteral final {
     std::span<const char> m_charactersWithNullTerminator;
 };
 
+using UnownedCString = ASCIILiteral;
```

Comment:
I don't think we want two classes that have different lifetime semantics to be aliases of each other.

Side point: if we are going to have a class for an unowned view into a CString, I think we should call it CStringView, for consistency will StringView, or NullTerminatedSpan, to state the missing fact. Also, it needs to document its lifetime semantics using LIFETIME_BOUND. (You can follow the example in StringView.h.)

Reply:
Yes, this alias is a bad idea. The point of having different classes is that they have different semantics, not that they are different names for the same underlying concept.